File: docs/guides/php/src/globalAlgoliaUserID.php
<?php

require __DIR__.'/../vendor/autoload.php';
use Algolia\AlgoliaSearch\Api\SearchClient;

$config = SearchConfig::create('ALGOLIA_APPLICATION_ID', 'ALGOLIA_API_KEY');
$config->setDefaultHeaders(['X-Algolia-UserToken' => 'test-user-123']);
$client = SearchClient::createWithConfig($config);
echo $client->getClientConfig()->getDefaultHeaders();

File: docs/guides/go/src/globalAlgoliaUserID.go
package main

import (
	"fmt"

	"github.com/algolia/algoliasearch-client-go/v4/algolia/next/search"
)

func globalAlgoliaUserID() {
	client, err := search.NewClient("YourApplicationID", "YourAdminAPIKey")
	if err != nil {
		fmt.Println(err)
	}

	client.AddDefaultHeader("X-Algolia-UserToken", "test-user-123")
}
